tools/shoestring/shoestring/wizard/setup_file_generator.py

```diff
@@ -129,3 +136,100 @@ def patch_shoestring_config(shoestring_filepath, package_config_filepath):
 	new_config_manager = ConfigurationManager(package_config_filepath.parent)
 	new_config_manager.patch(package_config_filepath.name, config_patches)
 	shutil.copy(package_config_filepath, shoestring_filepath)
+
+
+async def patch_bootstrap_shoestring_config(bootstrap_target_path, config_filepath, include_node_key=False):
+	"""Run shoestring import bootstrap command."""
+
+	with DisableLogger():
+		await run_import_bootstrap(ImportBootstrapArgs(config_filepath, bootstrap_target_path, include_node_key))
+
+
+def try_prepare_rest_overrides_file_from_bootstrap(screens, output_filename):  # pylint: disable=invalid-name
+	"""Prepares a REST overrides file based on the current rest.json."""
+
+	node_type = screens.get('node-type').current_value
+	bootstrap = screens.get('bootstrap')
+
+	if 'dual' != node_type:
```

Comment:
why are you only checking dual; does bootstrap not support api-only node?

Reply:
From the node-type screen, there is only dual, peer and light.  If dual node is selected and harvesting and voting is disable then it will create a api-only node.